tasks/RedundantRBSSelectorTask.cpp

```diff
@@ -194,6 +206,31 @@ void RedundantRBSSelectorTask::updateHook()
         _rbs_out.write(secondary_rbs);
     }
 }
+
+PoseDivergence RedundantRBSSelectorTask::checkDivergences(
```

Comment:
To optimize this method, we should use getEuler instead of getRoll, getPich and getYaw.
I forgot to do that when I implemented this